docs/using-hive.md

```diff
@@ -11,10 +11,6 @@
         ```bash
         oc create secret generic mycluster-pull-secret --from-file=.dockerconfigjson=/path/to/pull-secret --type=kubernetes.io/dockerconfigjson
         ```
-  1. Create a Kubernetes secret containing your AWS credentials:
-        ```bash
-        oc create secret generic test-creds --from-literal=aws_secret_access_key=$AWS_SECRET_ACCESS_KEY --from-literal=aws_access_key_id=$AWS_ACCESS_KEY_ID
-        ```
```

Comment:
We view create-cluster as a developer tool, and it might be good to keep this step documented here so external users of Hive's API get a clear picture of each step involved. Could we perhaps keep this block but add a NOTE indicating this will be done automatically if using hiveutil create-cluster.

Reply:
Make sense, thanks @dgoodwin